frontend/src/pages/PointsPage/components/PointDisplay/index.tsx

```diff
@@ -29,6 +29,9 @@ export function PointDisplay(props: PointProps): JSX.Element {
                       <Typography variant='h5' component='h2'>
                         {event.name}
                       </Typography>
+                      <Typography color='textSecondary'>
+                        {event.type} event
```

Comment:
I think it would be great if you can capitalize both `event.type` and the string `"event"` here. What do you think?